packages/taquito-local-forging/test/allTestsCases.ts

```diff
@@ -465,22 +468,22 @@ export const commonCases: TestCase[] = [
     },
   },
   // In `opMapping` from the file `constants.ts`, the operations and types starting at `chest` were added in the hangzhou protocol
-  ...extractOp(0, 140, opMapping).map((op) => {
+  ...extractOp(0, 140, opMapping).map((op): TestCase => {
     return {
       name: `Origination operation (${op})`,
       operation: {
         branch: 'BLzyjjHKEKMULtvkpSHxuZxx6ei6fpntH2BTkYZiLgs8zLVstvX',
         contents: [
           {
-            kind: 'origination',
+            kind: OpKind.ORIGINATION,
             counter: '1',
             source: 'tz1QZ6KY7d3BuZDT1d19dUxoQrtFPN2QJ3hn',
             fee: '10000',
             gas_limit: '10',
             storage_limit: '10',
             balance: '0',
             script: {
-              code: genericCode(op),
+              code: genericCode(op as unknown as MichelsonValue) as MichelsonV1Expression[],
```

Comment:
I think that genericCode should accept a string, see comment below. That way, it won't be necessary to caste here.